src/replica/backup/cold_backup_context.cpp

```diff
@@ -567,31 +573,31 @@ void cold_backup_context::on_upload_chkpt_dir()
     }
 
     if (checkpoint_files.size() <= 0) {
-        LOG_INFO("%s: checkpoint dir is empty, so upload is complete and just start write "
-                 "backup_metadata",
-                 name);
+        LOG_INFO_F("{}: checkpoint dir is empty, so upload is complete and just start write "
+                   "backup_metadata",
+                   name);
```

Comment:
```suggestion
        LOG_INFO_F("{}: checkpoint dir is empty, so upload is complete and just start write", name);
```

Reply:
"backup_metadata" is not an independent parameter, but a part of the first string parameter, so it should not be removed.